code/modules/antagonists/heretic/magic/aggressive_spread.dm

```diff
@@ -10,8 +10,8 @@
 	school = SCHOOL_FORBIDDEN
 	cooldown_time = 30 SECONDS
 
-	invocation = "A'GRSV SPR'D"
-	invocation_type = INVOCATION_WHISPER
+	invocation = "Agresiv'nyy rasprostra-neniye!"
```

Comment:
Not an expert in old slavic, but comparing to the russian, the ending on `Agresiv'nyy` assumes that the folowing noun is plural, but the ending of `rasprostra-neniye` is used for single forms.
Is this on purpose,  or a mistake?